src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4942,7 +4942,8 @@ private static bool CanReuseVariableDeclarator(CSharp.Syntax.VariableDeclaratorS
                             var expression = item as ExpressionSyntax;
                             if (expression != null)
                             {
-                                args.Add(_syntaxFactory.Argument(null, default(SyntaxToken), expression));
+                                args.Add(_syntaxFactory.Argument(null, default(SyntaxToken), expression,
+                                                                 type: null, identifier: default(SyntaxToken)));
```

Comment:
Seems a good place to use `Argument(ExpressionSyntax expression)` that was introduced in `SyntaxFactory`.